CMakeLists.txt

```diff
@@ -469,7 +469,10 @@ if(LUACHECK_EXECUTABLE)
     list(APPEND CHECK_QA_TARGETS luacheck)
 endif()
 add_custom_target(check-qa DEPENDS ${CHECK_QA_TARGETS})
-add_dependencies(check check-qa check-examples)
+add_dependencies(check check-qa)
+if(TARGET check-examples)
+    add_dependencies(check check-examples)
```

Comment:
I don't get why this change is necessary. Isn't check-examples a target anyway? https://github.com/Aire-One/awesome/blob/chore/bump-actions-cache-to-v4/tests/examples/CMakeLists.txt#L383

Reply:
This is a behavior change that is unexpected to me. `make check` should run all `check-*` targets.

Reply:
Are you sure that `check-examples` was run before? It was never included in this file thus was never run.